src/assets/css/twitterwall.css

```diff
@@ -42,6 +43,7 @@
 	
 }
 .container-tweet{
+	box-sizing: border-box;
 	width:504px;
 	height:240px;
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Fallback background-color (hex or RGB) should precede RGBA background-color.](https://www.codacy.com/app/triplez/open-event-webapp/file/2415102791/issues/source?bid=3233857&fileBranchId=3235518#l49)